src/main/resources/db/migration/common/V1_232__Update_TransactionUrgency_Table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE transcription_urgency ADD COLUMN IF NOT EXISTS display_state BOOLEAN NOT NULL DEFAULT TRUE;
+
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(4, 'Up to 3 working days');
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(5, 'Up to 7 working days');
+INSERT INTO darts.transcription_urgency (tru_id, description) VALUES(6, 'Up to 12 working days');
```

Comment:
These could be grouped

```suggestion
INSERT INTO
  darts.transcription_urgency (tru_id, description)
VALUES
  (4, 'Up to 3 working days'),
  (5, 'Up to 7 working days'),
  (6, 'Up to 12 working days');
```

Reply:
Updated